templates/includes/header.html

```diff
@@ -2,7 +2,7 @@
 
 {% include 'includes/login-status.html' %}
 
-<header class="tna-header tna-header--accent" data-module="tna-header">
+<header class="tna-header tna-header--accent {{ classes }}" data-module="tna-header">
```

Comment:
Would it be easier to use a boolean for this switch to avoid adding more classes and overrides?

```
<header class="tna-header{% '' if noAccent else ' tna-header--accent' %}" data-module="tna-header">
```

Then we could use the simpler:

```
{% include 'includes/header.html' with noAccent=True %}
```

⚠️ (untested code) ⚠️

Reply:
I think thought that this would then end up with the black background, rather than the off-black one?

Reply:
Should it not be black? It will be eventually anyway when the new header is added. For ease then, let's keep your solution.